backend/src/apiserver/server/run_server_test.go

```diff
@@ -680,189 +1006,27 @@ func TestListRuns(t *testing.T) {
 		DisplayName:    "run1",
 		ServiceAccount: "pipeline-runner",
 		StorageState:   apiv2beta1.Run_AVAILABLE,
-		CreatedAt:      &timestamp.Timestamp{Seconds: 2},
-		ScheduledAt:    &timestamp.Timestamp{Seconds: 2},
+		CreatedAt:      &timestamp.Timestamp{Seconds: 5},
+		ScheduledAt:    &timestamp.Timestamp{Seconds: 5},
 		FinishedAt:     &timestamp.Timestamp{},
 		PipelineSource: &apiv2beta1.Run_PipelineSpec{
-			PipelineSpec: pipelineSpecStruct,
+			PipelineSpec: createdRun.GetPipelineSpec(),
 		},
 		RuntimeConfig: &apiv2beta1.RuntimeConfig{
 			PipelineRoot: "model-pipeline-root",
+			Parameters:   make(map[string]*structpb.Value, 0),
 		},
+		State: apiv2beta1.RuntimeState_PENDING,
 	}
 
-	listRunsResponse, err := server.ListRuns(nil, &apiv2beta1.ListRunsRequest{})
+	listRunsResponse, err := server.ListRuns(nil, &apiv2beta1.ListRunsRequest{
+		ExperimentId: experiment.UUID,
+	})
 	assert.Nil(t, err)
 	assert.Equal(t, 1, len(listRunsResponse.Runs))
 	assert.Equal(t, expectedRun, listRunsResponse.Runs[0])
 }
 
-func TestvalidateCreateRunRequestV1(t *testing.T) {
```

Comment:
ditto: on validation methods

Reply:
There is no longer need to validate the request. A part of it sits in `toModelRun` and the rest is in the `CreateRun`. This is to apply the same validation for v1 and v2 messages.